uat/README.md

```diff
@@ -0,0 +1,32 @@
+##User Acceptance Test
+UATs are defined under uat module. 
+UATs use aws-greengrass-testing-standalone as the test framework to run the tests.
+aws-greengrass-testing-standalone is pulled as maven dependency from GG maven repo. 
+You can add/update UATs at uat source. 
+The uat module generates a UAT artifact (nucleus-uat-artifact.jar) which is an executable jar meant to run the UATs.
+
+##Running UATs locally
+* UAT runs require the credentials for the AWS account you want to use.
+* Ensure credentials are available in the environment.
```

Comment:
Environment variables is not the only way to provide AWS credentials. Another way is AWS Profile.

Reply:
Following up to this comment I think it would be better to just add

```suggestion
* Ensure credentials are available either by setting them in environment variables or [configuring them in your aws cli profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
```